### ISSUE TITLE
Bounty for meeting publicity

### ISSUE THREAD

**Issue by VanCardui** [SingularityNET-Archive/SingularityNET-Archive#89]
In our Q1 2024 budget, we have recognised the need to publicise our meetings better, and have allocated $20 per meeting as a bounty for someone to share and publicise them and encourage people to come. 

They can decide how they do it, within certain expectations of amount of effort made. Could comprise outreach to the actual archives and recordkeeping world; and to the wider crypto and AI world.  $20 per meeting - ideally the same person each time, for continuity.

After the date of opening this task, there will be 4 meetings remaining in Q1 - but we might not get an applicant immediately, so we will test this process on 3 meetings only.

**Comment by stephen-rowan** [comment]
Create Bounty on Dework - @stephen-rowan

**Comment by stephen-rowan** [comment]
Archive festival ! or event possible in Q2 2024 ?
In the meantime wait for Slate to do his magic.

**Comment by VanCardui** [comment]
Vanessa and Slate had a quick meeting on 14th Feb to map out the work: see https://docs.google.com/document/d/1pAK0cUzrwjTHJ4Ve6_bh-GlzC3VkZE4zAJb02ZMSYYU/edit?usp=sharing

Note - Slate mentioned that maybe a problem is the fortnightly meeting cadence (people don't know which week it is on). At the moment we don't have the resources or capacity to move to weekly, but maybe something to consider in the long term.

**Comment by VanCardui** [comment]
Slate posted a meeting invite in Archives WG's Discord channel - see https://discord.com/channels/909843832491896832/1150014944721514496/1209690119481860157.

There were new people in the Archives WG meeting on 21st Feb 2024, but most of them were here because they are doing a bounty, or because they heard about the meeting elsewhere. 

There was a blocker in that that he didn't have permissions to post links in Discord - Peter has resolved it for the Archives WG's channel, but maybe it also needs resolving for one or two other channels - perhaps Ambassador General? so that a meeting announcement can be posted a bit earlier for the next meeting on 6th March

**Comment by stephen-rowan** [comment]
Slate still not able to post in Ambassadors general Discord channel (Vani neither).
Not sure Discord channel raises interest. 

Invites sent in DMs also useful.
Main motivation is from interest in tools and document subject.
A more personal approach approaching people individually.

**Comment by Asad272002** [comment]
Yup i texted everyone personally & successfully providing them with the
date and time also told them that their contributions and participation in
the meeting were really valuable so we would love to have them in the next
meeting as well, it was a general greeting/invitation to the meeting.

I also contacted Peter about the links, he said that he has already given
me permission not sure why I am still not able to post but will try to
resolve this as well.
The reason this is occurring is probably due to the last discord hack that
happened that's why the embedding of links is not allowed.

Thanks for the recap .

**Comment by stephen-rowan** [comment]
New participants have found information on Archive WG via Discord

**Comment by VanCardui** [comment]
Complete as of 1st April 2024
